.github/workflows/ci.yml

```
@@ -0,0 +1,96 @@
+      uses: actions/upload-artifact@v4
+      with:
+        name: coverage-reports
+        path: |
+          backend/coverage/lcov.info
+          frontend/coverage/lcov.info
+
+  sonarcloud:
+    needs: test  # Depends on test job completing first
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)